examples/baseline-app/docker-compose.yml

```diff
@@ -0,0 +1,631 @@
+      - NATS_CLIENT_PREFIX=alice-nchain-consumer
+      - NATS_URL=nats://alice-nats:4222
+      - NATS_STREAMING_URL=nats://alice-nats-streaming:4222
+      - NATS_TOKEN=testtoken
+      - REDIS_HOSTS=alice-redis:6379
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://alice-nchain:8080/status"] # FIXME
+      interval: 1m
+      timeout: 1s
+      retries: 2
+      start_period: 10s
+    hostname: alice-nchain-consumer
+    networks:
+      - alice
+    restart: always
+
+  alice-statsdaemon:
+    image: provide/nchain
+    entrypoint: ./ops/run_statsdaemon.sh
+    container_name: alice-statsdaemon
+    depends_on:
+      - alice-nats-streaming
+      - alice-postgres
+      - alice-redis
+    environment:
+      - DATABASE_HOST=alice-postgres
+      - DATABASE_NAME=nchain_dev
+      - DATABASE_USER=nchain
+      - DATABASE_PASSWORD=nchain
+      - LOG_LEVEL=DEBUG
+      - NATS_CLIENT_PREFIX=statsdaemon
+      - NATS_URL=nats://alice-nats:4222
+      - NATS_STREAMING_URL=nats://alice-nats-streaming:4222
+      - REDIS_HOSTS=alice-redis:6379
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://alice-nchain:8080/status"] # FIXME
+      interval: 1m
+      timeout: 1s
+      retries: 2
+      start_period: 10s
+    hostname: alice-statsdaemon
+    networks:
+      - alice
+    restart: always
+
+  alice-vault:
+    image: provide/vault
+    container_name: alice-vault
+    depends_on:
+      - alice-postgres
+    environment:
+      - DATABASE_HOST=alice-postgres
+      - DATABASE_NAME=vault_dev
+      - DATABASE_USER=vault
+      - DATABASE_PASSWORD=vault
+      - LOG_LEVEL=DEBUG
+      - PORT=8080
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://alice-vault:8080/status"]
+      interval: 1m
+      timeout: 1s
+      retries: 2
+      start_period: 10s
+    hostname: alice-vault
+    networks:
+      - alice
+    ports:
+      - 8082:8080
+    restart: always
+
+  alice-nethermind:
+    image: provide/nethermind
+    container_name: alice-nethermind
+    environment:
+      - NETHERMIND_CONFIG=baseline
+      - NETHERMIND_INITCONFIG_STATICNODESPATH=Data/static-nodes-baseline-hosts.json
+      - NETHERMIND_JSONRPCCONFIG_ENABLED=true
+      - NETHERMIND_JSONRPCCONFIG_HOST=0.0.0.0
+      - NETHERMIND_JSONRPCCONFIG_PORT=8522
+      - NETHERMIND_KEYSTORECONFIG_TESTNODEKEY=020102030405060708090a0b0c0d0e0f000102030405060708090a0b0c0d0e0f
+      - NETHERMIND_NETWORKCONFIG_DISCOVERYPORT=30222
+      - NETHERMIND_NETWORKCONFIG_P2PPORT=30222
+      - NETHERMIND_VAULTCONFIG_HOST=alice-vault:8082
+    healthcheck:
+      test: ["CMD", "/usr/local/bin/await_tcp.sh", "localhost:8522"]
+      interval: 1m
+      timeout: 1s
+      retries: 2
+      start_period: 10s
+    hostname: alice-nethermind
+    networks:
+      - alice
+      - bob
+    ports:
+      - 8522:8522
+      - 30222:30222
+    restart: always
+    volumes:
+      - ./ops/await_tcp.sh:/usr/local/bin/await_tcp.sh:cached
+
+  # bob
+
+  bob-postgres:
```

Comment:
What gets stored in postgres containers? Full merkle tree (which is partially represented on-chain)?

Reply:
Will make sure this is well-covered in the README but postgres stores the local off-chain state- it does not persist the merkle tree db.